src/platform/intel/cavs/platform.c

```diff
@@ -322,14 +153,6 @@ int platform_init(struct sof *sof)
 	if (ret < 0)
 		return ret;
 
-	/* init low latency single channel DW-DMA domain and scheduler */
-	sof->platform_dma_domain =
-		dma_single_chan_domain_init
-			(&sof->dma_info->dma_array[PLATFORM_DW_DMA_INDEX],
-			 PLATFORM_NUM_DW_DMACS,
-			 PLATFORM_DEFAULT_CLOCK);
-	scheduler_init_ll(sof->platform_dma_domain);
```

Comment:
this is removing functionality that isn't used by default and has a high, but not 100% chance of being broken. Still it's also possible at least in theory that someone is using it now, isn't it? So, it's probably a decision call - whether we want to deprecate it first and maybe disable by default or just remove it as you do it here

Reply:
@lyakh Good catch. I did go a bit back and forth on this. But in the end, cAVS2.5 IPC4 config is only made available as a opt-in release (with e.g. SOF2.6), so the usage at this point is very limited, so chances of somebody using it for something exotic like DMA scheduler domain, would seem much smaller than odds of winning in lottery. So I went ahead and removed.